website/docs/cloud/ci.md

```diff
@@ -74,31 +75,3 @@ chmod +x eas-build-on-success.sh
 
 ✅ Check:
 - Now, your builds should be automatically uploaded to Flashlight when they are successfuly completed on EAS with the selected profile.
-
-#### Advanced usage - use flashlight on production builds
-
-If you want to use Flashlight on production builds, you can extend the `eas-build-on-success.sh` script to upload the production build to Flashlight. To do so, you need to install `bundletool` to convert the `.aab` file to `.apk` file and you need to generate a `keystore` to sign the `.apk` file.
-
-Here's an example of script that does all of that:
-
-```bash
-if [ "$EAS_BUILD_PROFILE" = "production" ] && [ "$EAS_BUILD_PLATFORM" = "android" ]; then
```

Comment:
actually I think we probably need to keep this check
and have the 2 if condition